### PR TITLE
New route to see a recipe by its identifier

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

**Comment by tsop14** [comment]
What happens if two people open the same recipe? The first person will not realise all their changes will be overwritten. Do we want to handle this?

**Comment by LATaylor-guardian** [comment]
@shtukas If you can view the recipe like this are you able to edit it also? Also, whats the reason for adding this endpoint - can you update the PR description with it please.

**Comment by shtukas** [comment]
@tsop14 
So... as discussed, this is currently a private route of us (admins) to access a recipe we have the id of. Not meant to be a part of the user curation workflow.

**Comment by shtukas** [comment]
> What happens if two people open the same recipe?
As discussed, let me remove the edit button [Submit Recipe] on that page (and the [Skip this recipe] button as well).

**Comment by shtukas** [comment]
Done. The buttons now only show up on the standard curation page, and not on the new "view only" page :)

**Comment by tsop14** [comment]
This could be useful to view articles that are consistently skipped?

👍

File: ui/app/com/gu/recipeasy/controllers/Application.scala [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
         val curatedRecipe = CuratedRecipe.fromRecipe(r)
         val curatedRecipeForm = CuratedRecipeForm.toForm(curatedRecipe)
         db.setRecipeStatus(r.id, "Pending")
-        Ok(views.html.recipe(createCuratedRecipeForm.fill(curatedRecipeForm), r.id, r.body, r.articleId))
+        Ok(views.html.recipe(createCuratedRecipeForm.fill(curatedRecipeForm), r.id, r.body, r.articleId, true))
```
Comment (LATaylor-guardian) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: For readability this would be better written as: 
Ok(views.html.recipe(createCuratedRecipeForm.fill(curatedRecipeForm), r.id, r.body, r.articleId, shouldShowButtons = true))

File: ui/app/com/gu/recipeasy/controllers/Application.scala [hunk 2]
```diff
@@ -54,6 +54,20 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
...
   }
+
+  def viewRecipe(recipeId: String) = Action { implicit request =>
+    val newRecipe = db.getRecipe(recipeId)
+    newRecipe match {
+      case Some(r) => {
+        val curatedRecipe = CuratedRecipe.fromRecipe(r)
+        val curatedRecipeForm = CuratedRecipeForm.toForm(curatedRecipe)
+        db.setRecipeStatus(r.id, "Pending")
+        Ok(views.html.recipe(createCuratedRecipeForm.fill(curatedRecipeForm), r.id, r.body, r.articleId, false))
```
Comment (LATaylor-guardian) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: As above.

File: ui/app/com/gu/recipeasy/controllers/Application.scala (hunk 1 above)
Comment (shtukas) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: Cool, will send a separate PR.
